proxy/src/http/source.rs

```diff
@@ -101,10 +101,14 @@ fn branches_filter(
     path("branches")
         .and(warp::get())
         .and(http::with_peer(peer))
-        .and(document::param::<String>(
+        .and(document::param::<coco::Urn>(
             "project_id",
             "ID of the project the blob is part of",
         ))
+        .and(warp::filters::query::query::<PeerQuery>())
```

Comment:
Shouldn't it be called `BranchesQuery`? Since it's the query string for `/branches`?

Reply:
Ya fair :+1: